src/ol/proj/Projection.js

```diff
@@ -3,6 +3,14 @@
  */
 import {METERS_PER_UNIT} from './Units.js';
 
+/**
+ * The function is called with a `number` view resolution and a
+ * {@link module:ol/coordinate~Coordinate} as arguments, and returns the `number` resolution
+ * in projection units at the passed coordinate.
+ * @typedef {function(number, import("../coordinate.js").Coordinate):number} GetPointResolution
+ * @api
+ */
```

Comment:
I'm not sure that this documentation is rendered or if the `@api` annotation has any effect. Do you see any difference in [the generated docs](https://deploy-preview-16481--ol-site.netlify.app/en/latest/apidoc/module-ol_proj_projection-projection) without it?

Reply:
Erp.  I missed this https://deploy-preview-16481--ol-site.netlify.app/en/latest/apidoc/module-ol_proj_projection#~GetPointResolution